charts/akeyless-gateway/values.yaml

```diff
@@ -113,6 +113,42 @@ globalConfig:
         cpu: 250m
         memory: 256Mi
 
+cacheHA:
+  enabled: false
+## TODO: Need to find the right value to control the Redis port
```

Comment:
what with the ###TODO?

Reply:
This is due to a gap with port overwrite, can be deleted for now